.github/workflows/website-deploy.yml

```diff
@@ -5,11 +5,6 @@ on:
       - '^v[0-9]+\.[0-9]+(\.[0-9]+)*'
     branches:
       - master
-    paths:
```

Comment:
The [documentation ](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) mention that `Path filters are not evaluated for pushes of tags.`  so the site will be deployed upon a tag regardless if the site was changed... am I missing something?